FIx verdaccio auth in npm-publish container

```
npm-publish_1            | npm ERR! code ENEEDAUTH
npm-publish_1            | npm ERR! need auth This command requires you to be logged in to http://verdaccio:4873/
npm-publish_1            | npm ERR! need auth You need to authorize this machine using `npm adduser`
```

--- a/infrastructure/development/containers/npm-build/build-up.sh
+++ b/infrastructure/development/containers/npm-build/build-up.sh
@@ -2,6 +2,9 @@
 
 REGISTRY_URL=http://verdaccio:4873/
 
+# Ping registry before build
+npm ping --registry http://verdaccio:4873/ || exit 1
+
 # Install packages using the local registry
 npm install --registry $REGISTRY_URL
 npm audit fix --registry $REGISTRY_URL

--- a/infrastructure/development/containers/npm-publish/publish-all-packages.sh
+++ b/infrastructure/development/containers/npm-publish/publish-all-packages.sh
@@ -3,12 +3,15 @@
 # Indicate that monitoring server is up
 HEALTH_CHECK_SERVER=0
 
+# Ping registry before authorization
+npm ping --registry http://verdaccio:4873/ || exit 1
+
 # NPM user credentials
 USERNAME=local-admin
 PASSWORD=root
 EMAIL=admin@local.npm
 
-# Authorize local repository account
+# Authorize local repository account (or exit)
 /usr/bin/expect <<EOD
 spawn npm adduser --registry http://verdaccio:4873/
 expect {

--- a/infrastructure/development/containers/npm-start/install-and-start.sh
+++ b/infrastructure/development/containers/npm-start/install-and-start.sh
@@ -2,6 +2,9 @@
 
 REGISTRY_URL=http://verdaccio:4873/
 
+# Ping registry before start
+npm ping --registry http://verdaccio:4873/ || exit 1
+
 # Current entrypoint script args
 NPM_START_ARGS=$@
 

